jira_sync/repositories/base.py

```diff
@@ -196,7 +162,128 @@ def get_open_issues(self) -> list[Issue]:
 
                 issues.extend(self.normalize_issue(issue) for issue in partial_issues)
                 next_page = self.get_next_page(endpoint="issues", response=response, **kwargs)
+                page += 1
```

Comment:
Now you mention it, it almost looks like it would make the code cleaner and simpler if I used the “walrus operator” (assignment expression, [since Python 3.8](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions)) instead of having the first call to `.get_next_page()` outside of the loop and counting pages:

```
response = None

while next_page := self.get_next_page(endpoint="issues", response=response, **kwargs):
    response = requests.get(**next_page)
    if response.status_code == requests.codes.ok:
        ...
    elif response is None and response.status_code == requests.codes.not_found:
        ...
```